torchrl/modules/planners/cem.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+
+from torchrl.data.tensordict.tensordict import TensorDictBase
+from torchrl.envs import EnvBase
+from torchrl.modules.planners import MPCPlannerBase
+
+__all__ = ["CEMPlanner"]
+
+
+class CEMPlanner(MPCPlannerBase):
+    """
+    CEMPlanner Module. This class inherits from TensorDictModule.
+
+    Provided a TensorDict, this module will perform a CEM planning step.
```

Comment:
Let's flip this

`This module will perform a CEM planning step when given a TensorDict containing XX and YY`